config.yaml

```
@@ -0,0 +1,23 @@
+app:
+  name: go-fitbyte
+
+server:
+  port: 3000
+  host: "localhost"
+  prefork: false
+
+# database:
+#   username: "abc"
+#   password: "password"
+#   name: "db-fitbyte-dev"
+#   host: "localhost"
+#   ports: 5432
+#   maxConnections: 10
+
+database:
+  username: "postgres"
+  password: "rootpass"
+  name: "mydb"
+  host: "localhost"
+  ports: 5432
+  maxConnections: 10
```

database di devcontainer salah host

```suggestion
database:
  username: "abc"
  password: "password"
  name: "db-fitbyte-dev"
  host: "db"
  ports: 5432
  maxConnections: 10
```